migrations/mysql/README.rst

```diff
@@ -2,6 +2,8 @@
 csv2json.py
 ===========
 
+**Attention: This tool works only with MySQL CSV dumps**
```

Comment:
This tools is only intended for migrating MySQL CSV dumps to Crate readable JSON (single JSON per row).